packages/valory/skills/market_manager_abci/bets.py

```diff
@@ -58,6 +68,32 @@ class Bet:
     status: BetStatus = BetStatus.UNPROCESSED
     blacklist_expiration: float = -1
 
+    @classmethod
+    def from_gamma_subgraph(cls, **kwargs: Any) -> "Bet":
+        """Initialize a bet's instance from gamma subgraph's attributes."""
```

Comment:
TODO: ignore empty ids as they are closed markets.

Reply:
They are not closed markets, but apparently not-yet updated on the subgraph. Replace the query as discussed.

Reply:
e76eca95c1606593207652ac723041ace5f17996